net/httping/patches/004-fix-cmake.patch

```diff
@@ -0,0 +1,32 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
```

Comment:
Why do we need to have this patch in our repository? This is not part of upstream repository.
Did you even tried to upstream it?